components/MetaData.js

```diff
@@ -14,19 +14,39 @@ export default function MetaData(props) {
         <meta name="keywords" content={d.keywords} />
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <link rel="icon" href="/favicon.ico" />
-        <meta name="dcterms.title" content="My Service Canada Account" />
-        <meta name="dcterms.language" content="eng" />
-        <meta name="dcterms.creator" content="Service Canada" />
-        <meta name="dcterms.accessRights" content="2" />
+        <meta
+          name="dcterms.title"
+          content={
+            props.language === 'en'
+              ? 'My Service Canada Account - My dashboard'
+              : 'Mon dossier Service Canada - Mon tableau de bord'
```

Comment:
This changes all the pages meta data to use "- my dashboard". I'd recommend passing in page related information using the component props

Reply:
props.language, dcterms.creator and dcterms.service will continue to be manually coded instead of pulling from AEM.